CHANGELOG.rst

```diff
@@ -8,6 +8,186 @@
 
 .. towncrier release notes start
 
+Pytest 3.3.0 (2017-11-23)
+=========================
+
+Deprecations and Removals
+-------------------------
+
+- Pytest no longer supports Python **2.6** and **3.3**. Those Python versions
+  are EOL for some time now and incur maintenance and compatibility costs on
+  the pytest core team, and following up with the rest of the community we
+  decided that they will no longer be supported starting on this version. Users
+  which still require those versions should pin pytest to ``<3.3``. (`#2812
+  <https://github.com/pytest-dev/pytest/issues/2812>`_)
+
+- Remove internal ``_preloadplugins()`` function. This removal is part of the
+  ``pytest_namespace()`` hook deprecation. (`#2236
+  <https://github.com/pytest-dev/pytest/issues/2236>`_)
```

Comment:
This link is wrong, it should be #2636 if I'm not mistaken...

Reply:
@uSpike it doesn't really matter, GH automatically redirects it to the appropriate page. 😉 

All of our links use the `pytest/issues/#` form, even if the number actually points to a PR.

Reply:
I'm confused because if I go to https://github.com/pytest-dev/pytest/issues/2236 it's a PR about junitxml, "Change junitxml.py to produce results that comply with Junitxml schema"

Reply:
@uSpike OH you are right, sorry I misinterpreted what you said.

I will fix this on `master` after the release, the package has just now finished testing and I would not like to go over the entire process again. 😁